File: app/src/main/java/dev/costas/vertext/viewmodels/ContentViewModel.java
package dev.costas.vertext.viewmodels;

import androidx.lifecycle.ViewModel;

public class ContentViewModel extends ViewModel {
	private String title = "";
	private String content =  "";

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}
}

File: app/src/main/kotlin/dev/costas/vertext/viewmodels/ContentViewModel.kt
package dev.costas.vertext.viewmodels

import androidx.compose.runtime.getValue
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.setValue
import androidx.lifecycle.ViewModel

class ContentViewModel : ViewModel() {
    var title by mutableStateOf("")
    var content by mutableStateOf("")
}